ethpm_cli/commands/registry.py

```diff
@@ -107,6 +112,25 @@ def activate_registry(uri_or_alias: str, config: Config) -> None:
         write_store_data_to_disk(activated_store_data, store_path)
 
 
+def explore_registry(uri: str, config: Config) -> None:
+    if is_valid_registry_uri(URI(uri)):
+        (
+            address,
+            chain_id,
+            pkg_name,
+            pkg_version,
+            namespaced_asset,
+            ens,
+        ) = parse_registry_uri(uri)
+        config.w3.enable_unstable_package_management_api()
+        config.w3.pm.set_registry(address)
+        package_names = config.w3.pm.get_all_package_names()
+        display_packages(package_names, config)
+
+    else:
+        raise UriNotSupportedError("Cannot resolve both an alias and registry uri.")
```

Comment:
I'm not sure I understand this error message. It seems as though we currently aren't accepting an alias? Also, I tend to use f-strings a lot in my error messages - displaying the incorrect variable in the mesage can be extremely useful for users and yourself later on.
Something more like...
```python
else:
    raise UriNotSupportedError(f"Invalid URI: {uri}, unable to explore. Registry URI definition can be found at https://docs.ethpm.com/uris#registry-uris.")
```

Reply:
See the other comment below but I think it's okay to get rid of this if we are going to use existing error messages?

Reply:
Yep, that looks like it should be fine